boards/nxp/imx95_evk/board.cmake

```diff
@@ -3,3 +3,12 @@
 if (CONFIG_SOF AND CONFIG_BOARD_IMX95_EVK_MIMX9596_M7_DDR)
   board_set_rimage_target(imx95)
 endif()
+
```

Comment:
we need add document on how to use spsdk on imx95.

Reply:
No problem. Thanks.